packages/experiments/src/components/Tile/Tile.tsx

```diff
@@ -168,7 +169,7 @@ export class Tile extends React.Component<ITileProps, ITileState> {
 
     const { isSelected = false, isModal = false } = this.state;
 
-    const isSelectable = !!selection && selectionIndex > -1;
+    const isSelectable = isItemSelectable || (!!selection && selectionIndex > -1);
```

Comment:
This should be:
```ts
const {
  isSelectable = !!selection && selectionIndex > -1
} = this.props;
```
Basically, prefer the new value, falling back to the old value if `props.isSelectable` is `undefined`.

Reply:
@ThomasMichon  updated PR based on suggestion :)